client/src/Dashboard/DashboardTitle.js

```diff
@@ -96,21 +103,67 @@ export default function DashboardTitle({ dates, userState, getDashboardData }) {
     </Dropdown>
   )
 
+  const addPayment = async () => {
+    const paymentsBatch = Object.entries(childPayments).flatMap(data => {
+      return {
+        month: lastMonth.toISOString().split('T')[0],
+        amount: data[1],
+        child_id: data[0]
+      }
+    })
+
+    const response = await makeRequest({
+      type: 'post',
+      url: '/api/v1/payments_batches',
+      headers: {
+        Authorization: token
+      },
+      data: {
+        payments_batch: paymentsBatch
+      }
+    })
+
+    if (response.ok) {
+      setPaymentModalVisible(false)
+    } else {
+      // TODO: handle bad request
+      console.log(response, 'bad request')
+    }
+  }
   const paymentModal = (
     <Modal
-      id="paymentModal"
-      title={
-        <div className="eyebrow-large text-center">{t('recordAPayment')}</div>
-      }
+      className="payment-modal"
+      title={<div className="text-center h2-large">{t('recordAPayment')}</div>}
+      closeIcon={<CloseOutlined className="-btn-primary" />}
       visible={isPaymentModalVisible}
       onCancel={handlePaymentModalCancel}
+      //todo determine width. Maybe 50% of screen size
```

Comment:
I can fix this in #1623 as I will be playing around with sizing there

Reply:
Hi @davidcandeau @katelovescode yes, the width of the modal is 2/3 the width of the screen. We use a 24-column grid and the modal is 16 columns.